app/_hub/kong-inc/acl/how-to/_consumer-groups.md.md

```diff
@@ -0,0 +1,93 @@
+---
+nav_title: Using ACLs with consumer groups
+title: Using ACLs with consumer groups
+---
+
+## ACL configuration with {{site.base_gateway}}
+
+This example covers a common use case: as an API owner, you want to regulate access based on the type of request methods and consumer groups. Specifically, the goal is to allow consumers in the dev group to perform GET, POST, and PUT requests on all routes, while reserving the DELETE request functionality exclusively for consumers in the admin group.
+
+
+### Create consumer groups 
+
+1. Using the API, create a consumer group for `dev`:
+     ```bash
+     curl --request POST \
+         --url http://localhost:8001/consumer_groups \
+         --header 'Content-Type: application/json' \
+         --header 'accept: application/json' \
+         --data '{"name":"dev"}'
+     ```
+
+1. Then create a consumer group for `admin`:
+     
+     ```bash
+     curl --request POST \
+       --url http://localhost:8001/consumer_groups \
+       --header 'Content-Type: application/json' \
+       --header 'accept: application/json' \
+       --data '{"name":"admin",}'
+     ```
+
+1. Add a consumer to the `admin` group by using the UUID of the specific consumer:
```

Comment:
```suggestion
1. Using the UUID of an existing consumer, add a consumer to the `admin` group:
```
Wait - you need to have a consumer before you do this? This step doesn't _create_ a consumer, correct? Only adds an existing one? This wasn't mentioned anywhere in prerequisites. Can go one of two ways with this: add steps to create, then add a consumer, or just say you need to have consumers available in a prerequisites section.